src/main/java/io/netty/incubator/codec/http3/Http3RequestStreamInboundHandler.java

```diff
@@ -109,6 +125,26 @@ protected void channelRead(@SuppressWarnings("unused") ChannelHandlerContext ctx
         frame.release();
     }
 
+    /**
+     * Called once a {@link QuicException} should be handled. By default, these exceptions are just logged
+     *
+     * @param ctx           the {@link ChannelHandlerContext} of this handler.
+     * @param exception     the {@link QuicException} that caused the error.
+     */
+    protected void handleQuicException(@SuppressWarnings("unused") ChannelHandlerContext ctx, QuicException exception) {
+        logger.error("Caught QuicException.", exception);
```

Comment:
Since the pipeline appropriately handles `QuicException` and no user action is typically required, we can log at `DEBUG` level here.

Reply:
Also can you add the channel to the log message here? It will help debugging.